sound/soc/sof/intel/hda-loader-skl.c

```diff
@@ -484,7 +489,7 @@ cl_skl_cldma_copy_to_buf(struct snd_sof_dev *sdev, const void *bin,
 			cl_skl_cldma_fill_buffer(sdev, bufsize, bufsize,
 						 curr_pos, true);
 
-			ret = cl_skl_cldma_wait_interruptible(sdev);
+			ret = cl_skl_cldma_wait_interruptible(sdev, false);
```

Comment:
Will we ever wait for any use case ? If not, then we can just remove the wait function ?

Reply:
If only support polling mode is OK, then we can remove the wait interrupt function.